CommitChange/houdini
# Remove most ramda from nonprofits

incomplete (some ramda left here still, mostly more complicated stuff)
split out from https://github.com/CommitChange/houdini/pull/964

to test
- dashboard
  - shows active events
  - shows payments chart, can change dates, monthly/daily/etc., click on legend to change what's shown
- payments > recurring tab > click stats, should show
- supporters
  - should be able to merge two supporters
  - should be able to import supporters
  - should be able to create a supporter
  - choose a supporter
    - make sure activities show as expected
    - try creating, editing, and deleting a note
- make a donation, make sure all forms / steps work as expected
